public/app/angular/services/UtilSrv.ts

```diff
@@ -0,0 +1,64 @@
+import { GrafanaRootScope } from 'app/angular/GrafanaCtrl';
+import { HideModalEvent, ShowModalEvent } from '../../types/events';
+import { deprecationWarning } from '@grafana/data';
+import { appEvents } from 'app/core/app_events';
+
+/**
+ * Old legacy utilSrv exposed to angular services and handles angular modals.
+ * Not used by any core or known external plugin.
+ */
+export class UtilSrv {
```

Comment:
nit: maybe we should just deprecate the whole class?

Reply:
I don't think it matters, any usage of this class is not using this type anyway (either using utilSrv injected via angular dependency injection, and we have no types for this, never been exposed via typed lib), but more likely angular plugins are likely using it via the appEvents.